<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartPage</class>
 <widget class="QWidget" name="StartPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1450</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="gameNameLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>300</y>
     <width>360</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Wilderness Quest</string>
   </property>
  </widget>
  <widget class="QPushButton" name="instructionsButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Instructions</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>350</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boyScoutButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Boy Scout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="girlScoutButton">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>290</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Girl Scout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nonBinaryScoutButton">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>290</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Non-Binary Scout</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
